README.md

```diff
@@ -97,35 +97,22 @@ To view the documentation including examples, contracts, component status, and t
 
 1. `git clone https://github.com/OfficeDev/office-ui-fabric-react.git`
 2. `cd office-ui-fabric-react`
-3. `npm install`
-4. `npm start`
+3. `yarn`
+4. `yarn builddemo`
+5. `yarn start`
 
 This will start a demo page from the office-ui-fabric-react package folder, which will open a web browser with the example page. You can make changes to the code which will automatically build and refresh the page using live-reload.
 
-To build and run tests for all packages in the repo, run `npm run build` from the root.
+To build and run tests for all packages in the repo, run `yarn build` from the root.
 
-To build individual packages within the `packages/*` or `apps/*` folders, `cd` to the relevant folder and run `npm run build`. Note that because the packages are symlinked together, you must manage building dependencies in the right order, or use the `rush` tool to build to the specific package you want. (See advanced tips below.)
+To build individual packages within the `packages/*` or `apps/*` folders, `cd` to the relevant folder and run `yarn build`. Note that because the packages are symlinked together, you must manage building dependencies in the right order, or use the `yarn build` script to build to the specific package you want. (See advanced tips below.)

```

Comment:
I think it should say `yarn buildto` towards the end of the paragraph, not `yarn build`? Also probably remove the part about the advanced tips since now it's really right below this paragraph.
```suggestion
To build individual packages within the `packages/*` or `apps/*` folders, `cd` to the relevant folder and run `yarn build`. Note that because the packages are symlinked together, you must manage building dependencies in the right order, or use the `yarn buildto` script to build to the specific package you want.
```